kombu/transport/redis.py

```diff
@@ -146,8 +146,14 @@ def __init__(self, *args, **kwargs):
     def append(self, message, delivery_tag):
         delivery = message.delivery_info
         EX, RK = delivery['exchange'], delivery['routing_key']
+        if redis.VERSION[0] >= 3:
+            # Redis-py changed the format of zadd args in v3.0.0
+            zadd_args = [{time(): delivery_tag})
```

Comment:
Oh , this syntax might only work on py3?

Reply:
You missed a `]` and replaced it with `)`.

Reply:
I suspect that you may have the arguments reversed here. redis-py 3.0 expects the dict to be a mapping where element names are keys and scores are their values. I believe this should be:

```python
zadd_args = [{delivery_tag: time()}]
```

Reply:
Yeah, `zadd_args = [{delivery_tag: time()}]` was what I meant to do. Sorry, I did this via the github editor :( Fixing it now.

Reply:
Fixed.